libraries/abstractions/wifi/include/iot_wifi.h

```diff
@@ -65,6 +65,7 @@ typedef enum
     eWiFiSecurityWEP,         /**< WEP Security. */
     eWiFiSecurityWPA,         /**< WPA Security. */
     eWiFiSecurityWPA2,        /**< WPA2 Security. */
+    eWiFiSecurityWPA2_ent,    /**< WPA2 Enterprise Security. */
```

Comment:
Hi, @WenChieh-Feng .
I can understand having the above enum, but there's no code that makes use of this new enum.  Using this enum in a config may result in non-working wifi.

Reply:
Also, where is the client configuration for WPA2-Enterprise configured and stored?  That isn't in this PR.

Reply:
Hi @gkwicker  - This PR is only for the changes that are required in the common interface (iot_wifi.h) to support enterprise security feature on Cypress platform. The actual implementation will be part of the code which will be under vendors/cypress folder, that will available in our upcoming release